tests/ltp/Makefile

```diff
@@ -20,10 +20,34 @@ ifndef FS
 FS=ext2fs
 endif
 
+# *** start fork related settings ***
+# fork is now compiled into the code all the time and is controled
+# through config. As LTP needs fork to function we now need to 
+# use the fork enabled ltp tests, and we need to use the config.json
+# to use the fork implementation
+
+#original ltp source
+# URL=https://github.com/linux-test-project/ltp
+# BRANCH=master
+
+# fork enabled
+URL=https://github.com/paulcallen/ltp
+BRANCH=paulcallen/revert-fork
+
+# Fork disabled
+#URL=https://github.com/mikbras/ltp
+#BRANCH=myst
+
 TEST_FILE_NAME_EXTENSION=
-ifdef MYST_PSEUDO_FORK
-TEST_FILE_NAME_EXTENSION=fork.
-endif
+#ifdef MYST_PSEUDO_FORK
+#TEST_FILE_NAME_EXTENSION=fork.
+#endif
+
+#ifdef MYST_PSEUDO_FORK
+OPTS += --app-config-path config.json
```

Comment:
Without `MYST_PSEUDO_FORK`, applications can still take a ForkMode option but just ignore it.

Reply:
I have removed all references to MYST_PSEUDO_FORK define as the fork code is always compiled in, but is disabled by default.